### DISCUSSION TITLE
Ability to filter local addresses out

### DISCUSSION THREAD

**Discussion by jdtate101** [umami-software/umami#2552] [General]
### Describe the feature or enhancement

Can you add some sort of filter to exclude certain IP addresses or ranges from the stats? I'm thinking about all local IP ranges. At the moment they get recognised as UNKNOWN, but still count in the visitor and hit totals, having them excluded would just show real visitors.

**Comment by Maxime-J** [comment]
There's a runtime environment variable you could use
`IGNORE_IP = a comma-delimited list of IP addresses or ranges`

For example `IGNORE_IP = 10.0.1.0/24,192.168.1.200`

 Ranges are not mentioned in the docs (https://umami.is/docs/environment-variables) but they are supported.

**Reply by jlar0che** [reply]
Thanks for the info. After IP Addresses and/or ranges are added into the runtime environment variable is there a way to verify they are actively being ignored within the Umami interface? In other words, is there some visual confirmation within an Umami instance showing what IPs are being ignored?

This would be very helpful -- especially if they could be tagged with info (like xxx.xxx.xxx.xxx = Michael's Home IP).
